.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules/
 /npm-debug.log
+*.compiled.js
```

Comment:
EOF newline, please. :)

See [Newline at End of File Support][1] if you'd like your editor to insert this automatically.


[1]: https://github.com/editorconfig/editorconfig/wiki/Newline-at-End-of-File-Support